internal/compiler/tests/syntax/imports/library.slint

```diff
@@ -8,8 +8,6 @@ import { NotFound } from "@not-found/foo.slint";
 
 import { NotFound2 } from "@test-lib/lib.slint";
 //                        ^error{No exported type called 'NotFound2' found in "@test-lib/lib.slint"}
-import { NotFound3 } from "@test-lib/not-existing.slint";
-//                        ^error{Cannot find requested import "@test-lib/not-existing.slint" in the library search path}
```

Comment:
The new error contains the OS error that is not cross platform, so I just remove this test. This is already covered by the `test_library_import_errors` anyway.